apps/dashboard/app/javascript/files/data_table.js

```diff
@@ -378,14 +377,21 @@ class DataTable {
     }
 
     renderNameColumn(data, type, row, meta) {
-        const cssClass = `${row.type} name ${row.type == 'd' ? '' : 'view-file'}`;
```

Comment:
I think these classes were only used in testing. I don't see any `d`, `f` or `view-file` as a real CSS class, only `name` is used somewhere as a selector (not as an actual CSS class).

Type has been moved from a CSS class to `data-type` for querying.